### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by pefoley2**

### CODE REVIEW

**Comment by pefoley2** [comment]
@jwoglom: ping

File: intranet/apps/printing/views.py [hunk 1]
```diff
@@ -108,6 +108,30 @@ def convert_file(tmpfile_name):
...
+                if not len(rr) == 2:  # make sure 2 values in range
+                    return False
+                else:
+                    if not(0 < int(rr[0]) < max_pages) and not(0 < int(rr[1]) < max_pages):  # check in page range
+                        return False
+                    if (int(rr[0]) > int(rr[1])):   # check lower bound <= upper bound
+                        return False
+                    pages += int(rr[1]) - int(rr[0]) + 1
+            else:
+                if not (0 < int(r) <= max_pages):  # check in page range
```
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517841174/issues/source?bid=3176160&fileBranchId=3273604#l127)

File: intranet/apps/printing/views.py [hunk 2]
```diff
@@ -108,6 +108,30 @@ def convert_file(tmpfile_name):
...
+    """ Returns the number of pages in the range, or False if it is an invalid range. """
+    pages = 0
+    try:
+        for r in page_range.split(","):  # check all ranges separated by commas
+            if "-" in r:
+                rr = r.split("-")
+                if not len(rr) == 2:  # make sure 2 values in range
+                    return False
+                else:
+                    if not(0 < int(rr[0]) < max_pages) and not(0 < int(rr[1]) < max_pages):  # check in page range
```
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517841174/issues/source?bid=3176160&fileBranchId=3273604#l121)

File: intranet/apps/printing/views.py [hunk 3]
```diff
@@ -108,6 +108,30 @@ def convert_file(tmpfile_name):
...
+                        return False
+                    if (int(rr[0]) > int(rr[1])):   # check lower bound <= upper bound
```
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'if' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517841174/issues/source?bid=3176160&fileBranchId=3273604#l123)

File: intranet/apps/printing/views.py (hunk 2 above)
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517932153/issues/source?bid=3176160&fileBranchId=3273604#l121)

File: intranet/apps/printing/views.py (hunk 1 above)
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'not' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517932153/issues/source?bid=3176160&fileBranchId=3273604#l127)

File: intranet/apps/printing/views.py (hunk 3 above)
Comment (tjhsst-ion) [inline intranet/apps/printing/views.py]: Codacy Issue found: [Unnecessary parens after 'if' keyword](https://www.codacy.com/app/tjcsl/ion/file/2517932153/issues/source?bid=3176160&fileBranchId=3273604#l123)

**Pull request merged**